artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/ha/ColocatedHAManager.java

```diff
@@ -163,6 +174,23 @@ private synchronized boolean activateSharedStoreBackup(String journalDirectory,
       return true;
    }
 
+   private TopologyMember validateBackupGroupName(SimpleString nodeID) {
+      // Older versions of artemis don't send the nodeID in BackupRequestMessage:
+      // in this case we cannot trust the request, making the requesting server
```

Comment:
Why can't we, we know we had old servers and setups working fine, as such why be so strict, simply fail back (or even tter have a v1 and v2 version of the message, that way if all v2 cluster you can be fully strict, else in a v1/v2 mix you can be less strict

Reply:
It was working fine because you haven't had any other member of topology with a group name and been lucky, I believe :P